samples/synchronization/README.rst

```diff
@@ -44,3 +49,33 @@ Sample Output
    <repeats endlessly>
 
 Exit QEMU by pressing :kbd:`CTRL+A` :kbd:`x`.
+
+Usage for UART tracing backend
+*******************************
+Build image with "cmake -DBOARD=sam_e70_xplained -DCONF_FILE=prj_uart.conf .."
+
+After the board runnig a while, running script trace_capture_uart.py with
```

Comment:
How about:
```
After the application has run for a while, run the
``trace_capture_uart.py`` script on your host::

   sudo python3 trace_capture_uart.py -d /dev/ttyACM1 -b 115200 -o log_1

The sam_e70_xplained's serial port is ``/dev/ttyACM1``
and is used as the tracing backend. The script's output file
``log_1`` and metadata under ``zephyr/subsys/debug/tracing/ctf/tsdl``
can be used to serve Tracecompass.
```

Where is Tracecompass described?

Reply:
@dbkinder many thanks, updated, also the Tracecompass is here https://www.eclipse.org/tracecompass/